Ileriayo/banka
# #165554922 Add bcrypt and jwt

#### What does this PR do?
- Protects routes from unauthorized users

#### Description of Task to be completed?
- Install bcrypt ```npm i -D bcryptjs``` to hash passwords that will be  stored in the database

#### How should this be manually tested?
- Install jwt ```npm i -D bcryptjs``` to sign and verify tokens to protect routes
- After this, test coverage will drop significantly

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- [#165554922](https://www.pivotaltracker.com/story/show/165554922
![Capture](https://user-images.githubusercontent.com/13166712/56752248-bf546f00-677f-11e9-81ab-26838be30bc5.PNG)
) 

#### Screenshots (if appropriate)
![Capture](https://user-images.githubusercontent.com/13166712/56752276-cc715e00-677f-11e9-878e-46631c8c93ec.PNG)
